Earthfile

```diff
@@ -740,6 +740,21 @@ ipxe-iso:
     SAVE ARTIFACT /build/ipxe/src/bin/ipxe.iso iso AS LOCAL build/${ISO_NAME}-ipxe.iso
     SAVE ARTIFACT /build/ipxe/src/bin/ipxe.usb usb AS LOCAL build/${ISO_NAME}-ipxe-usb.img
 
+raw-image:
```

Comment:
What can this raw image be used for? Since this target is not used anywhere else, consider adding a comment to explain where this is useful.

Reply:
just a raw disk image with the recovery, same used on cloud. Will add a comment pointing to the docs